feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -129,6 +129,26 @@ func (s *RelayRouter) getSessionProxyClient(
 	return protos.NewCentralSessionControllerClient(conn), ctx, cancel, nil
 }
 
+//N7_proxy service for 5G
+func (s *RelayRouter) getN7ProxyClient(
+	c context.Context, imsi string) (protos.CentralSessionControllerClient, context.Context, context.CancelFunc, error) {
+	conn, ctx, cancel, err := s.GetFegServiceConnection(c, imsi, FegN7Proxy)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	return protos.NewCentralSessionControllerClient(conn), ctx, cancel, nil
+}
+
+//Service selection based on the RAT Type
+func (s *RelayRouter) getProxyClient(
```

Comment:
move this function before the other two. This way the reader can read incrementally